core/integration/features/liquidity-provision/liquidity_provision_simple_pdp.feature

```diff
@@ -37,21 +37,22 @@ Feature: Test LP orders
       | party            | market id | side | volume | price | resulting trades | type       | tif     | reference       |
       | sellSideProvider | ETH/DEC19 | sell | 100000 | 120   | 0                | TYPE_LIMIT | TIF_GTC | sell-provider-1 |
       | buySideProvider  | ETH/DEC19 | buy  | 100000 | 80    | 0                | TYPE_LIMIT | TIF_GTC | buy-provider-1  |
-      | party1           | ETH/DEC19 | buy  | 50000  | 110   | 0                | TYPE_LIMIT | TIF_GTC | lp-ref-1        |
-      | party1           | ETH/DEC19 | sell | 50000  | 120   | 0                | TYPE_LIMIT | TIF_GTC | lp-ref-2        |
-    Then the orders should have the following states:
-      | party            | market id | side | volume | price | status        |
-      | sellSideProvider | ETH/DEC19 | sell | 100000 | 120   | STATUS_ACTIVE |
-      | buySideProvider  | ETH/DEC19 | buy  | 100000 | 80    | STATUS_ACTIVE |
+      | party1           | ETH/DEC19 | buy  | 500    | 110   | 0                | TYPE_LIMIT | TIF_GTC | lp-ref-1        |
+      | party1           | ETH/DEC19 | sell | 500    | 120   | 0                | TYPE_LIMIT | TIF_GTC | lp-ref-2        |
+
+    Then the market data for the market "ETH/DEC19" should be:
+      | static mid price | best static bid price | best static offer price | 
+      |              115 |                   110 |                     120 |
+
     Then the parties submit the following liquidity provision:
       | id  | party  | market id | commitment amount | fee | side | pegged reference | proportion | offset | lp type    |
-      | lp1 | party1 | ETH/DEC19 | 50000             | 0.1 | buy  | BID              | 500        | 10     | submission |
-      | lp1 | party1 | ETH/DEC19 | 50000             | 0.1 | sell | ASK              | 500        | 10     | submission |
+      | lp2 | party1 | ETH/DEC19 | 50000             | 0.1 | buy  | BID              | 500        | 10     | submission |
+      | lp2 | party1 | ETH/DEC19 | 50000             | 0.1 | sell | ASK              | 500        | 10     | submission |
     Then the liquidity provisions should have the following states:
       | id  | party  | market    | commitment amount | status        |
-      | lp1 | party1 | ETH/DEC19 | 50000             | STATUS_ACTIVE |
+      | lp2 | party1 | ETH/DEC19 | 50000             | STATUS_ACTIVE |
     
     Then the orders should have the following states:
-      | party  | market id | side | volume | price | status        |
-      | party1 | ETH/DEC19 | buy  | 225000 | 100   | STATUS_ACTIVE |
-      | party1 | ETH/DEC19 | sell | 153847 | 130   | STATUS_ACTIVE |
+      | party  | market id | side | volume | price | status        | reference |
+      | party1 | ETH/DEC19 | buy  |  49450 |   100 | STATUS_ACTIVE | lp2       |
```

Comment:
would the LP buy vol not be 50000/100=500, and then convert to position decimal, 50000?
would the LP sell vol not be 50000/130=385, and then convert to position decimal, 38500?

Reply:
I think this is correct. The static volume of `party1` is `500 @ 110 (buy)` and `500 @ 120 (sell)`. So your formula above shouldn't apply. Let's take buy side: the static volume `party1` is providing `5 x 110` so the remaining volume to deploy is `(50000 - 550)/100 = 494.5` which is exactly what we see.